src/Common/tests/System/Diagnostics/RemoteExecutorConsoleApp/RemoteExecutorConsoleApp.csproj

```diff
@@ -25,5 +25,10 @@
       <Link>Common\System\Diagnostics\CodeAnalysis\ExcludeFromCodeCoverageAttribute.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup>
+    <Content Include="$(OutDir)\$(AssemblyName).runtimeconfig.json">
```

Comment:
It's feels weird to have this project include an output file here. Can we inject this `Content` item in the target above, instead?